### PR TITLE
Fix tests for non ostree build

### PR DESCRIPTION
**Pull request by eu-smirnov**

### CODE REVIEW

File: src/libaktualizr/uptane/uptane_test.cc [hunk 1]
```diff
@@ -477,6 +477,7 @@ TEST(Uptane, ProvisionOnServer) {
 TEST(Uptane, FsToSqlFull) {
   TemporaryDirectory temp_dir;
   Utils::copyDir("tests/test_data/prov", temp_dir.Path());
+  chmod(temp_dir.Path().c_str(), S_IRWXU);
```
Comment (patrickvacek) [inline src/libaktualizr/uptane/uptane_test.cc]: Why don't we just do this automatically in `copyDir`? I think it's safe to say that we always want to enforce the same specific permissions.

File: CMakeLists.txt [hunk 2]
```diff
@@ -136,7 +136,7 @@ if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE Valgrind)
 endif(NOT CMAKE_BUILD_TYPE)
 
-set(CMAKE_CXX_FLAGS_DEBUG "-g")
+set(CMAKE_CXX_FLAGS_DEBUG "-O0 -g")
```
Comment (patrickvacek) [inline CMakeLists.txt]: How about `-0g`? I forget if clang supports that, but g++ definitely does, and it helps keep down the bloat a bit.

File: src/libaktualizr/uptane/uptane_test.cc (hunk 1 above)
Comment (eu-smirnov) [reply src/libaktualizr/uptane/uptane_test.cc]: I think it's better to do it explicitly. Also, usual `copy` behavior is to copy permissions from the source. So if we replace `Utils::copyDir` with `copy_directory` from boost or C++14 this line will be needed.

File: CMakeLists.txt (hunk 2 above)
Comment (eu-smirnov) [reply CMakeLists.txt]: Thanks, that will be better. Clang also has it since 4.0.

File: src/libaktualizr/uptane/uptane_test.cc (hunk 1 above)
Comment (patrickvacek) [reply src/libaktualizr/uptane/uptane_test.cc]: Fair enough, that sounds reasonable.

**Pull request merged**